fetch.bs

```diff
@@ -7095,11 +7095,11 @@ descendants share a quota of 384 kibibytes. That value is computed as such:
 
 <p class=note>This is called on navigation, when the source document of the navigation is the
 <a for=/>navigable</a>'s parent document. It potentially reserves either 64kb or 8kb of quota for
-the frame, if  the permissions policy allow. It is not observable to the cotnainer document whether
-the reserved quota was used in practice. This algorithm assumes that the container's document might
-delegate quota to the navigated frame, and the reserved quota would only apply in that case, and
-would be ignored if it ends up being shared. If quota was reserved and the document ends up being
-<a>same origin</a> with its parent, the quota would be
+the container and its navigable, if allowed permissions policy. It is not observable to the
+cotnainer document whether the reserved quota was used in practice. This algorithm assumes that the
+container's document might delegate quota to the navigated frame, and the reserved quota would only
```

Comment:
> might delegate quota to the navigated frame

what's `frame` here? Is it container's navigable?

Reply:
Container, rewording